src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -24,59 +83,82 @@ const ArtistSeriesHeader: React.FC<ArtistSeriesHeaderProps> = props => {
 
 const ArtistSeriesHeaderLarge: React.FC<ArtistSeriesHeaderProps> = props => {
   const {
-    artistSeries: { title, description },
+    artistSeries: { title, description, artists },
   } = props
   return (
-    <Box>
-      <Grid>
-        <Row>
-          <Col sm={6}>
-            <Flex
-              height="100%"
-              flexDirection="column"
-              justifyContent="space-between"
-            >
-              <Sans size="8" element="h1" unstable_trackIn>
-                {title}
-              </Sans>
-              <Sans lineHeight={1.5} size="3t" pr={1}>
-                {description}
-              </Sans>
-            </Flex>
-          </Col>
-          <Col sm={6}>
-            <Box
-              // FIXME: Use Image
-              bg="black10"
-              width={1}
-              height={400}
-            ></Box>
-          </Col>
-        </Row>
-      </Grid>
-    </Box>
+    <>
+      <Flex
+        position="relative"
+        alignItems="center"
+        justifyContent="center"
+        p={2}
+      >
+        <Flex position="absolute" left={3}>
+          {artists.length > 0 ? <ArtistInfo artist={artists[0]} /> : null}
```

Comment:
Quick trick! Instead of this ternary, you can just do `{ artists.length && <ArtistInfo...`